api/src/test/java/org/apache/iceberg/types/TestCharSeqComparator.java

```diff
@@ -52,18 +53,20 @@ public void testSeqLength() {
     Comparator<CharSequence> cmp = Literal.of(s1).comparator();
 
     // Sanity check that String.compareTo gives the same result
-    Assert.assertTrue(
-        "When one string is a substring of the other, the longer is greater", s1.compareTo(s2) < 0);
-    Assert.assertTrue(
-        "When one string is a substring of the other, the longer is greater", s2.compareTo(s1) > 0);
+    assertThat(s1)
+        .as("When one string is a substring of the other, the longer is greater")
+        .isLessThan(s2);
```

Comment:
can we use the same assertion check in `TestBinaryComparator`?

Reply:
Here also the method under test is `Comparator.compare`